conf/metrics.properties.template

```diff
@@ -192,4 +201,10 @@
 
 #driver.source.jvm.class=org.apache.spark.metrics.source.JvmSource
 
-#executor.source.jvm.class=org.apache.spark.metrics.source.JvmSource
\ No newline at end of file
+#executor.source.jvm.class=org.apache.spark.metrics.source.JvmSource
+
+# Example configuration for PrometheusServlet
+#*.sink.prometheusServlet.class=org.apache.spark.metrics.sink.PrometheusServlet
+#*.sink.prometheusServlet.path=/metrics/prometheus
+#master.sink.prometheusServlet.path=/metrics/master/prometheus
+#applications.sink.prometheusServlet.path=/metrics/applications/prometheus
```

Comment:
@dongjoon-hyun Will this swamp the Prometheus server? A lot of metrics of numerous Spark applications will be sent to the central server, for example, `metrics_{$app_id}_driver_BlockManager_disk_diskSpaceUsed_MB_Value 0`

Have you tried to run it ? How to deal with the high load issues?

Reply:
Actually, I've use this in a long-running spark streaming app in the dedicated Spark standalone cluster.

Reply:
And, I fully understand what you mean.

Reply:
@gatorsmile . Did you try to use `spark.metrics.namespace`?
```
$ bin/spark-shell --conf spark.metrics.namespace=ns
... (running)
```

```
$ curl -s http://localhost:4040/metrics/prometheus/ | grep driver_BlockManager_disk_diskSpaceUse
metrics_ns_driver_BlockManager_disk_diskSpaceUsed_MB_Value 0
```

Reply:
The key is generated by `MetricsSystem.buildRegistryName`, not by the individual `Sink`s.
- http://spark.apache.org/docs/latest/monitoring.html#metrics
> However, often times, users want to be able to track the metrics across apps for driver and executors, which is hard to do with application ID (i.e. spark.app.id) since it changes with every invocation of the app. For such use cases, a custom namespace can be specified for metrics reporting using spark.metrics.namespace configuration property. 

I'd like to improve this further according to the additional requirement. Please let me know.